src/components/DpDatepicker/DpDatepicker.stories.mdx

```diff
@@ -0,0 +1,18 @@
+import { Meta } from '@storybook/addon-docs'
+import DpDatepicker from './'
+
+<Meta
+    title="Components/Datepicker"
+    component={DpDatepicker}
+/>
+
+# Datepicker
+
+The Datepicker component use `a11y-datepicker` component.
```

Comment:
The first paragraph of the documentation of each component should express these things:

- what the component does (aka. what the component enables users to do)
- when to use the component (a helpful question here would be: for which occasions do we use the component)

A good example can be found in the DpDetails documentation.

I think we do not have to mention that `a11y-datepicker` is used internally, as this is an implementation detail that users of `DpDatepicker` do not have to care about.

Reply:
Done in https://github.com/demos-europe/demosplan-ui/pull/521/commits/1c48425c019ec7a0de619a4b3ccf39b2a36a0925